.editorconfig

```diff
@@ -135,7 +135,7 @@ csharp_new_line_between_query_expression_clauses = true
 csharp_indent_block_contents = true
 csharp_indent_braces = false
 csharp_indent_case_contents = true
-csharp_indent_case_contents_when_block = true
+csharp_indent_case_contents_when_block = false
```

Comment:
We changed this to `false` in dotnet/runtime sometime back. This is just making it consistent so we don't end up with unnecessary extra indentation for case statements.

Reply:
I'll put up a separate PR normalizing the other code after this goes in.

For those unaware, VS defaults to:
```csharp
switch (n)
{
    case 0:
        break;

    case 1:
        {
            break;
        }
}
```

`csharp_indent_case_contents_when_block = false` allows for, which matches the more typical usage for blocks:
```csharp
switch (n)
{
    case 0:
        break;

    case 1:
    {
        break;
    }
}
```

We've not changed the default in VS due to historical back-compat, but many of our repos have explicitly opted into the "better" formatting, so they aren't penalized for using braces

Reply:
Not going to change this in the PR after all. The repo apparently has some different configuration setup for the editorconfig and code formatting and while none of the errors are reporting in VS, they are reporting in the CI build even when marked as `silent`